maven/dev-workflow-ui-screenshots/pom.xml

```diff
@@ -14,21 +14,13 @@
     <relativePath>../config</relativePath>
   </parent>
 
-  <distributionManagement>
```

Comment:
Can you really remove this?

Reply:
think so, it's also defined in the parent, i.e. dev-workflow-ui-config